src/status_im/ui/screens/discover/recent_statuses/views.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.ui.screens.discover.recent-statuses.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require
+    [re-frame.core :refer [subscribe dispatch]]
+    [status-im.components.react :refer [view
```

Comment:
no refer, since react is so common I think `r` is OK, but check other (recent) namespaces or clojure chan for where we landed on this

Reply:
Never mind it being a common ns, just think how markup will look like. I literally stopped with namespace refer refactor when I saw `[react/view ....` maybe `[r/view ]` is a bit better, I don't know.

Reply:
It's not that bad, and IIRC we agreed to use namespaced fns a while ago. It gets rid of a class of bugs and it makes it clear where thing comes from - useful in a reasonably big code base such as ours.

See https://github.com/status-im/status-react/blob/develop/src/status_im/ui/screens/wallet/main/views.cljs#L22 for recent example